patches/third_party-lottie-lottie_worker.js.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/lottie/lottie_worker.js b/third_party/lottie/lottie_worker.js
+index adb3ff4793654284dc3e5790a140dfd97e0cc8a6..eb4d2b6a9d2ffda790fea840e91f9786002c7ed4 100644
+--- a/third_party/lottie/lottie_worker.js
++++ b/third_party/lottie/lottie_worker.js
+@@ -12033,6 +12033,7 @@ function initAnimation(animationData, initParams, canvas) {
+     }
+   });
+ 
++  currentAnimation.addEventListener("complete", () => postMessage({ name: 'complete' }));
```

Comment:
Just to clarify, we need to this event because the `initialized` event only fires on success?

Reply:
Lottie animations have iterations of run. On every iteration of the animation the service worker reports "complete" event. We need this event to be able to perform UI related tasks after an animation completes. 

In the new Welcome UI experience we have an animation that communicates the user's setup has completed. After the animation completes, we change the UI. Here's an [example ](https://lottiefiles.com/52093-complete)of what that animation looks like.